core/blockchain.go

```diff
@@ -775,31 +775,37 @@ func (self *BlockChain) reorg(oldBlock, newBlock *types.Block) error {
 }
 
 func (self *BlockChain) update() {
-	events := self.eventMux.Subscribe(queueEvent{})
+	// Subscribe to blockchain insertion event sets
+	eventCh := self.eventMux.Subscribe(queueEvent{}).Chan()
```

Comment:
Once up on a time there were more events but this is no longer the case. We don't need the overhead of the event machine because the `queueEvent` is no longer needed anywhere else.

Remove the overhead of the event machine and move to a function / method instead.